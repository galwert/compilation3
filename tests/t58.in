int getInput() {
    return 1; // TODO - Actually read input from user
}

void main(){
    int x = getInput();
    int y = getInput();
    int XplusY = x + y;
    int XdivY = getInput() / y;
    bool XeqY = x == y;

    if (XeqY)
    {
        print("I was expecting that");
    }
    
    bool XgtY = x > y;
    bool XltY = x < y;
    bool xneY = x != y;




    if (x != y) {
        return;
    }

    x = getInput();
    y = getInput();

    XgtY = x >= y;
    XltY = x <= y;
    XgtY = x > y;
    XltY = x < y;

    XneY = not XeqY; // XneY wasn't defined (Capital X here)
    XeqY = (not XgtY) and (not XltY);
    if (((not XgtY) and (not XltY)) or (not (not (not XeqY))))
    {
        print("\r\nEPIC\n\n");
    }


}