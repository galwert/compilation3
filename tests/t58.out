---end scope---
---end scope---
---end scope---
---end scope---
---end scope---
line 36: variable XneY is not defined
